pkg/uevent/indexer_test.go

```
@@ -80,10 +86,14 @@ func TestListDrives(t *testing.T) {
 	d1 := createTestDrive("test-node", "D1", "d9877501-e1b5-4bac-b73f-178b29974ed5")
 	d2 := createTestDrive("test-node", "D2", "b9475609-e1b5-4986-vs33-178131rdes9")
 	var drives []*directcsi.DirectCSIDrive
+	var err error
 	drives = append(drives, d1, d2)
 	indexer := createFakeIndexer()
-	indexer.store.Add(d1)
-	indexer.store.Add(d2)
+	_ = indexer.store.Add(d1)
```

can we not ignore the error here?

---

we can , but `golint ./... ` was complaining so i added the same .

---

can you try 

```
if err := indexer.store.Add(d2); err != nil {
            t.Errorf("error while adding objects to store: %v", err)
}
```